### PR TITLE
Transaction Tests

### PR DESCRIPTION
**Pull request by winsvega**
A few changes

### CODE REVIEW

File: test/TestHelper.cpp [hunk 1]
```diff
@@ -170,7 +170,7 @@ void ImportTest::importTransaction(json_spirit::mObject& _o)
 	}
 }
 
-void ImportTest::exportTest(bytes _output, State& _statePost)
+void ImportTest::exportTest(bytes const& _output, State const& _statePost)
```
Comment (CJentzsch) [inline test/TestHelper.cpp]: you need to change the definition of the function in TestHelper.h too.

Comment (winsvega) [inline test/TestHelper.cpp]: done

**Comment by winsvega** [comment]
Added some new changes to the test cases

File: test/block.cpp [hunk 2]
```diff
@@ -506,9 +508,9 @@ BOOST_AUTO_TEST_CASE(blInvalidHeaderTest)
 	dev::test::executeTests("blInvalidHeaderTest", "/BlockTests", dev::test::doBlockTests);
 }
 
-BOOST_AUTO_TEST_CASE(userDefinedFileBl)
+BOOST_AUTO_TEST_CASE(userDefinedFile)
 {
-	dev::test::userDefinedTest("--bltest", dev::test::doBlockTests);
+	dev::test::userDefinedTest("--test", dev::test::doBlockTests);
```
Comment (CJentzsch) [inline test/block.cpp]: please change --test to --singletest (also for the other test suites)

**Comment by gavofyork** [comment]
needs rebasing.

**Pull request merged**